app/controllers/v1/applications_controller.rb

```diff
@@ -12,4 +12,10 @@ def create
 
     # respond_with @legalaid_application
   end
+
+  def show
+    render json: Applicant.joins(:legal_aid_application)
```

Comment:
If you’ve got an application GET I’d expect it to select the Application and left join an Applicant.  This is assuming the applicant is assigned after the Application is created.

I might also expect the applicant to be a object which is returned on an application object. E.g.

```
{
  id: 2,
  applicant: {
    name: “Adrian”
  }
}
```